### PR TITLE
Add devise

### PR DESCRIPTION
**Pull request by Martum**
### Summary
- Add Devise gem and basic configuration.
- Add User model with migration.

### Trello card

https://trello.com/c/EMqPUxTP/9-instalar-configurar-devise-y-crear-user

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -35,6 +35,8 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'devise'
```
Comment (francoprud) [inline Gemfile]: It will be awesome if you add a specific version to the gem with a brief description

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'bc214d7894bdd0f26cdaddc4533ebb9254fd1765471061e6c732ac2f152e93b680c42e642e19d8e55fb26d294bd2a63fd83b6a5f4772f9f56a9c0f80be9003cf'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (francoprud) [inline config/initializers/devise.rb]: :thinking: Add this to a secrets variable

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
```
Comment (francoprud) [inline app/models/user.rb]: Add the presence validations to `first_name` and `last_name`

File: config/secrets.yml [hunk 4]
```diff
@@ -12,11 +12,14 @@
 
 development:
   secret_key_base: 212c3d67c1c1017b8a6497f7ab7a5837a725a1d444cf95e96bb6ff9a9f9acc719f62f4558aecd7dd6b2cf0f3167c2def3d75305dd6de9e0fb75bb290e2894ab4
+  devise_sender_email: "zarasa@gmail.com"
```
Comment (francoprud) [inline config/secrets.yml]: Hahahaha it can be a _no-reply@wbooksapi.com.ar_, don't you think?

Comment (Martum) [inline config/secrets.yml]: hahah sure!

**Pull request merged**